src/utils/validators.js

```diff
@@ -110,6 +114,19 @@ export const validateAmount = ({
         return MIN_ACCOUNT_BALANCE > BigInt(accountBalance) - BigInt(amountInBase);
       },
     },
+    FEE_RANGE: {
+      message: i18n.t(
+        `Fee must be greater than ${convertFromBaseDenom(
+          minValue,
+          token
+        )} and less than ${convertFromBaseDenom(accountBalance, token)}.`
+      ),
```

Comment:
shouldn't this be provided via interpolation

```suggestion
      message: i18n.t(
        `Fee must be greater than {{ maxFee }} and less than {{ minFee }}.`,
        {
          minFee: convertFromBaseDenom(accountBalance, token),
          maxFee: convertFromBaseDenom(
            minValue,
            token
          ),
        }
      ),
```